### DISCUSSION TITLE
more imaginary when increasing temperature

### DISCUSSION THREAD

**Discussion by kw600** [SSCHAcode/python-sscha#204] [General]
Dear SSCHA developers,

I am studying a CDW system using the experimental structure and lattice parameters. It is very werid that at the temperature proposed by experiments the phase 1 is unstable and increasing temperature makes it more unstable (however decreasing temperature to very low temperature, even at 0K, it becomes stable, which is different from unstable harmonic results). This is not the common way that temperature helps make the imaginary mode stable. When I look at the instability of phase 1 at high temperature, I find the instability is different from the harmonic instability, which has already been stabilized. If I follow this instability of phase 1 at high temperature, it gives a lower symmetry struture and performing the anharmonic calculations again gives the stable phase 2. We have checked the convergence of kpoints and q points and also the convergence of free energy hessian. Is there any possible reason increasing temperature makes the soft mode more unstable?

Best wishes,
Kang

**Comment by diegomartinez2** [comment]
Dear Kang,
Without knowing the system I can't give you a clear answer, we need more information. Are you looking to the SSCHA modes or the hessian ones?.

Diego

**Reply by diegomartinez2** [reply]
OK, I would be surprised if the SSCHA auxiliary modes are imaginary.

**Reply by kw600** [reply]
Many thanks for your reply. 
I am talking about the hessian. Yes, the auxiliary matrix is positively defined and I have considered both the d3 and d4 correction, which gives imaginary mode at high temperature but positive mode at low temperature.

**Reply by diegomartinez2** [reply]
In principle that means that there is a phase transition as Lorenzo says.

**Comment by mesonepigreco** [comment]
@kw600 Yes, it is possible. A typical case of this is melting: above the transition, your crystal structure becomes unstable as it wants to melt. Indeed also, different scenarios are possible. Usually, in these cases, it is not a second-order phase transition but a first-order one.

**Reply by kw600** [reply]
Hi Lorenzo, 
Many thanks for your help.

I agree the case of melting shows imaginary mode at high temperature and it helps me to understand it more. In my case, since the temperature range is from 200k to 600k so the material should still be a crystal. Could you give me an example of material and it undergoes the  first-order transition with the low temperature being stable while the high temeprature being unstable?

Best wishes,
Kang

**Reply by mesonepigreco** [reply]
This occurs in all first-order phase transitions, even between different crystalline phases. The low-temperature phase remains metastable even after the transition temperature to an upper critical temperature value, above which it becomes unstable and displays imaginary modes.

**Reply by mesonepigreco** [reply]
We studied something like this in high pressure hydrogen. In that case it was with pressure, not temperature, but we observed a structure that increasing pressure becomes harmonically more stable (phonons becomes harder) but when accounting for ZPM, there exist a pressure above which the phonons suddenly becomes unstable, even if the harmonic one are stable. The reason is that the system wants to undergo to a first order phase transition as I was mentioning.
